test/integration-new/ipamd/ipamd_test.go

```diff
@@ -0,0 +1,91 @@
+package ipamd
```

Comment:
The name of the test file could be more descriptive.

Reply:
Updated test file name